src/checkers/gnomechecker.py

```diff
@@ -16,6 +17,12 @@
 GNOME_MIRROR = URL("https://download.gnome.org/")
 
 
+class VersionScheme(StrEnum):
+    DEFAULT = "default"
+    ODD_MAJOR_IS_UNSTABLE = "odd-major-is-unstable"
```

Comment:
Are there any projects that follow this scheme?

Reply:
I been trying to come up with an example. It might be better to leave this out.

Reply:
I removed that enum value, I am not sure if we have relevant examples.

Reply:
It can be added back later if needed.